### PR TITLE
Added tech stack prototype

### PR DESCRIPTION
**Pull request by Nick-McGee**

### CODE REVIEW

**Comment by eloiseespel** [comment]
Very good research this week Nick! I like how you found out how the code builder works, I hope we get access to the source code so we can use that, looks way suited for our project

**Comment by eloiseespel** [comment]
I think it would be cool if we get to use some Python libraries or Azure cognitive services, I would certainly like to learn. Great to have it there as an option

File: techStack.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Currently, the tech stack is limited to what is compatible with Minecraft Education. This means that the primary languages used will be C++, Python, and JSON. The students will use Python in their lessons to learn AI/ML concepts. To create the lessons in Minecraft Education, we will use Everglade: an add-on to Minecraft Education’s Codebuilder used to teach Python. Lessons will be created with a JSON file and a Jupyter Notebook file.
+
+
+An example of Everglade in Minecraft Education. https://meecode.blob.core.windows.net/everglade/ci/index.html?lesson=https://meecode.blob.core.windows.net/everglade/content/row/master/tutorials/tutorials/Agent_Build/Agent_Build.json
+
+
+Consider: Tensorflow/Pytorch/Keras; or Azure/Google Cloud/AWS?
```
Comment (Ansper11) [inline techStack.txt:7]: I think we may not need to have our own AI model in the game

Comment (yyht6300a) [reply techStack.txt:7]: Thanks for the example. That helps a lot

**Comment by eloiseespel** [comment]
Just about Azure Cognitive Services it might be tough to integrate with the game

**Comment by eloiseespel** [comment]
By that I just mean that it's meant for external use I'm not sure it would connect to our code but if it does it would be cool

**Pull request merged**